modules/tinymce/src/plugins/lists/test/ts/browser/BackspaceDeleteTest.ts

```diff
@@ -7,7 +7,7 @@ import Plugin from 'tinymce/plugins/lists/Plugin';
 
 describe('browser.tinymce.plugins.lists.BackspaceDeleteTest', () => {
   const hook = TinyHooks.bddSetupLight<Editor>({
-    plugins: 'lists',
+    plugins: 'lists accordion',
```

Comment:
Is `accordion` really required? This test seems like it would work without that. I don't like having cross-plugin dependencies, even if it's in tests.

Reply:
Fixed.